tests/Doctrine/Tests/ORM/Mapping/YamlMappingDriverTest.php

```diff
@@ -83,6 +84,22 @@ public function testSpacesShouldBeIgnoredWhenUseExplode(): void
         self::assertEquals(255, $nameField['length']);
         self::assertEquals(255, $valueField['length']);
     }
+
+    public function testCompositeKeyForJoinTableInManyToManyCreation(): void
```

Comment:
This is unrelated, but made tests fail due to PHP 8,0 and no nesting. Saw the original `DDC3711Test.php` where this came from re-executed all YamlMappingDriver tests *again*, so this skips about 50 tests here that were duplicated.